Tmain/utils.sh

```diff
@@ -12,6 +12,11 @@ remove_commit_id()
     sed -i -e '/!_TAG_PROGRAM_VERSION.*/s#/[^/]*/#//#' $1
 }
 
+filesize()
+{
+    wc -c "$1" | awk '{ print $1 }'
```

Comment:
Currently the `awk` is also removing the leading whitespace.

```
$ wc -c < test.rb
      56
```

I guess in my usage the whitespace doesn't matter, so I can remove the `awk`.